package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pdf-lib",
-  "version": "1.2.0",
```

Comment:
Would you please remove this change? I prefer to modify the version myself when I decide to cut a release. It makes life easier, especially if I end up batching multiple changes together in a single release.

Reply:
Yes, my bad. I added the change for my fork so I could have a noticeable difference for my fork.

It's been removed. 👍